Formula/pcl.rb

```diff
@@ -13,39 +12,34 @@ class Pcl < Formula
     sha256 "d14889b636e81d1427a7d0300c028b571ac038e54b3760eba8900a7175d210c1" => :high_sierra
   end
 
-  depends_on "cmake" => :build
+  depends_on "cmake" => [:build, :test]
   depends_on "pkg-config" => :build
   depends_on "boost"
   depends_on "cminpack"
   depends_on "eigen"
   depends_on "flann"
-  depends_on "glew"
+  depends_on "libomp"
+  depends_on "libpcap"
   depends_on "libusb"
   depends_on "qhull"
   depends_on "vtk"
 
-  # Upstream patch for boost 1.70.0
-  patch do
-    url "https://github.com/PointCloudLibrary/pcl/commit/648932bc.diff?full_index=1"
-    sha256 "23f2cced7786715c59b49a48e4037eb9dea9abee099c4c5c92d95a647636b5ec"
-  end
-
   def install
     args = std_cmake_args + %w[
       -DBUILD_SHARED_LIBS:BOOL=ON
       -DBUILD_apps=AUTO_OFF
       -DBUILD_apps_3d_rec_framework=AUTO_OFF
       -DBUILD_apps_cloud_composer=AUTO_OFF
       -DBUILD_apps_in_hand_scanner=AUTO_OFF
-      -DBUILD_apps_optronic_viewer=AUTO_OFF
       -DBUILD_apps_point_cloud_editor=AUTO_OFF
-      -DBUILD_examples:BOOL=ON
+      -DBUILD_examples:BOOL=OFF
       -DBUILD_global_tests:BOOL=OFF
       -DBUILD_outofcore:BOOL=AUTO_OFF
       -DBUILD_people:BOOL=AUTO_OFF
-      -DBUILD_simulation:BOOL=AUTO_OFF
+      -DBUILD_simulation:BOOL=OFF
       -DWITH_CUDA:BOOL=OFF
       -DWITH_DOCS:BOOL=OFF
+      -DWITH_OPENGL:BOOL=OFF
```

Comment:
You might be able to enable these again now. We merged some last minute fixes after realizing that OpenGL is still being shipped in Catalina.